SixTrack/sixtrack.s

```diff
@@ -18524,15 +18526,16 @@ c$$$         endif
       call spliterr(errno,nunit,lineno,nfields,nf,lf,chars)
       end
       
-      double precision function fround(errno,fields,f)
+      real(kind=fPrec) function fround(errno,fields,f)
+      use floatPrecision
       implicit none
       integer maxf
       ! MAXF be kept in sync with maxf in various routines
       ! We maybe should use len(field(f)) here, like is done in splitfld...
       parameter (maxf=30)
       integer errno,f
       character*(*) fields(*)
-      double precision round_near,value
+      real(kind=fPrec) round_near,value
       fround=round_near(errno,maxf,fields(f))
```

Comment:
Hmm... `SixTrack/roundctl/round_near.c::double round_near_(int*,int*char*)` does never return a quad... It will always returns a double.

Reply:
Yes, all these functions need updating